internals/search_fulltext.py

```diff
@@ -157,6 +161,29 @@ def index_feature(fe: FeatureEntry) -> None:
   updated_fw_list[0].put()
 
 
+def canonicalize_string(s: str) -> str:
+  """Return a string of lowercase words separated by single spaces."""
+  lower_s = s.lower().replace("'", "")
```

Comment:
Optional: maybe trim the white spaces using .strip() as well

Reply:
That's not needed because the next line `WORD_RE.findall()` finds words in the string regardless of any whitespace.